### PR TITLE
rust icon added

### PR DESCRIPTION
**Pull request by orhanbalci**
Here is my screenshot for rust icon

developer_rust_icon

### CODE REVIEW

**Comment by dritter** [comment]
Cool. Thx. Looks awesome 👍 

I think we should set the Image for `awesome-patched` mode to an empty string as well. Being able to use the original code point from devicons is the advantage of fontconfig, and is not possible on the other modes..

**Comment by orhanbalci** [comment]
Set empty icon fore `awesome-patched` mode.

**Comment by bhilburn** [comment]
@orhanbalci - Thanks so much for the great PR! We are really excited to get this upstream.

Two requests:
1. Can you add a commit that adds the Rust icon to the README so that users will know about it?
2. It looks like you may have inadvertently introduced whitespace on one line. I'll mark it, below:

File: functions/icons.zsh [hunk 1]
```diff
@@ -120,7 +121,8 @@ case $POWERLEVEL9K_MODE in
       VCS_REMOTE_BRANCH_ICON         ' '$'\UF204 '          # 
       VCS_GIT_ICON                   $'\UF113 '             # 
       VCS_HG_ICON                    $'\UF0C3 '             # 
-    )
+      RUST_ICON                      $'\UE6A8'              #  
+	  )
```
Comment (bhilburn) [inline functions/icons.zsh]: Can you bring this end-paren back to it's original spot? Thanks!

**Comment by bhilburn** [comment]
@orhanbalci - Looks awesome!! Merged! `=)` Thanks for the great contribution.

**Comment by orhanbalci** [comment]
@bhilburn I added a link to rust icon in README and fixed the whitespace

**Comment by orhanbalci** [comment]
@bhilburn thanks for feedback =)

**Pull request merged**